pkg/cli/cli.go

```diff
@@ -154,3 +158,41 @@ Load noobaa completion to bash:
 
 	return rootCmd
 }
+
+func viperSetup(flagsets ...*pflag.FlagSet) {
+	viper.SetConfigName("noobaa.cfg")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath("/etc/noobaa/")
+	viper.AddConfigPath("$HOME/.noobaa")
+	viper.AddConfigPath(".")
+
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
+			logrus.Warn("failed to read config:", err)
+		}
```

Comment:
I couldn't figure out when this can happen.

Reply:
I think this can happen when say the config file is not valid (valid as in not adhering to yaml spec) `yaml`. In such cases we would want just log that there was some issue in reading the file.